### PR TITLE
Fix #35

### PR DESCRIPTION
**Pull request by allevo**

### CODE REVIEW

File: example/public/index.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+
```
Comment (mcollina) [inline example/public/index.css]: can you remove those spaces?

**Comment by allevo** [comment]
Why does travis fail? On my local machine, `npm t` runs well
https://travis-ci.org/fastify/fastify-static/jobs/336973844
```
  error responses can be customized with fastify.setErrorHandler() > /../index.js
  not ok socket hang up
```

File: example/public/index.css (hunk 1 above)
Comment (allevo) [reply example/public/index.css]: Done

**Comment by mcollina** [comment]
try “npm update”.

**Comment by delvedor** [comment]
`rm -rf node_modules && npm i && npm test`

**Pull request merged**